src/gatling/conf/recorder.conf

```diff
@@ -0,0 +1,53 @@
+recorder {
```

Comment:
It looks like every property of this file is commented out, though the template stays. What's the point? (pure curiosity)

Reply:
Just so one knows where to config things. I am happy to remove them if you'd prefer ;)